csrc/scheduler/tools/resize_utils.cpp

```diff
@@ -72,7 +72,8 @@ void propagateResizeToInputs(Expr* resize_tensor_op) {
       continue;
     }
 
-    scheduler_tools::scheduleLoopDomainsBy(tvs_to_schedule, resize);
+    scheduler_tools::scheduleLoopDomainsBy(
+        tvs_to_schedule, resize, Direction::Forward);
```

Comment:
Added the direction option to make it more explicit as it's always forward transformations.